parametrization.md.html

```diff
@@ -72,6 +72,45 @@
 | `geometry_coat_normal`                | Coat Normal         | `vector3` | N/A               |               | unperturbed normal   |                  |
 | `geometry_coat_tangent`               | Coat Tangent        | `vector3` | N/A               |               | unperturbed tangent  |                  |
 
+
+## UI disabling logic
```

Comment:
Would there be a value in broadening the scope of this proposal, moving it to the Standard UI Attributes section of the MaterialX specification?  That might help to build consistency between implementations of OpenPBR Surface and implementations of other popular shading models such as Standard Surface and glTF PBR:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/documents/Specification/MaterialX.Specification.md#standard-ui-attributes

Reply:
Since OpenPBR is not MaterialX-specific, I don't think it would make sense to describe the logic only for MaterialX. It would be valuable to figure out how to express the described enabling/disabling logic in MaterialX form though (or if it currently isn't possible, to figure out how to make it possible).